Core/Steps/ContinueReleaseStep.cs

```diff
@@ -77,7 +77,7 @@ public void Execute (string? ancestor, bool noPush)
     }
     else
     {
-      throw new InvalidOperationException("You have to be on a prerelease/* or release/* branch to continue a release.");
+      throw new UserInteractionException("You have to be on a prerelease/* or release/* branch to continue a release.");
```

Comment:
I'm probably not 100% consistent either because it's hard to do in the review screen. Please check to use the same structure / naming rules.